src/main/java/com/conveyal/gtfs/GTFS.java

```diff
@@ -81,16 +81,24 @@ public static FeedLoadResult load (String filePath, DataSource dataSource) {
      *   1. The tables' id column has been modified to be auto-incrementing.
      *   2. Primary keys may be added to certain columns/tables.
      *   3. Additional editor-specific columns are added to certain tables.
-     * @param feedId        feed ID (schema namespace) to copy from
-     * @param dataSource    JDBC connection to existing database
-     * @return              FIXME should this be a separate SnapshotResult object?
+     * @param feedId                feed ID (schema namespace) to copy from
+     * @param dataSource            JDBC connection to existing database
+     * @param normalizeStopTimes    whether to normalize stop sequence values on snapshot
+     * @return the result of the snapshot
      */
-    public static SnapshotResult makeSnapshot (String feedId, DataSource dataSource) {
-        JdbcGtfsSnapshotter snapshotter = new JdbcGtfsSnapshotter(feedId, dataSource);
+    public static SnapshotResult makeSnapshot (String feedId, DataSource dataSource, boolean normalizeStopTimes) {
```

Comment:
Should this new parameter be renamed normalizeStopTimesSequence or something similar? that's what this parameter utlimately controls in the normalizeAndCloneStopTimes method.